iptables/iptables.go

```diff
@@ -0,0 +1,30 @@
+package iptables
+
+import (
+	"errors"
+	"os"
+
+	"github.com/coreos/go-iptables/iptables"
+)
+
+// AddRule adds the required rule to the host's nat table
+func AddRule(appPort, metadataAddress, hostInterface string) error {
+	hostIP := os.Getenv("HOST_IP")
```

Comment:
Is `HOST_IP` something set by Kubernetes automatically? I'd rather not mix flags and environment variables, so I would prefer to move this to a flag like the others, it would be trivial to use the env var to set the flag e.g. `---host-ip=$HOST_IP`.

If I were to mix env vars and flags, I'd probably go with [viper](https://github.com/spf13/viper) but this is beyond what should be done in this PR, so let's stick with flag for now.